bundles/com.espressif.idf.core/src/com/espressif/idf/core/util/IDFUtil.java

```diff
@@ -993,4 +994,29 @@ public static String runEspDetectConfigScript()
 			return null;
 		}
 	}
+	
+	/**
+	 * Remove the variables which can affect the idf_tools.py export command
+	 * These variables can come from system environment
+	 * @param environment
+	 */
+	public static void cleanUpSystemEnvironment(Map<String, String> environment)
+	{
+		List<String> keysToRenmove = new LinkedList<>();
+		keysToRenmove.add(IDFEnvironmentVariables.IDF_PYTHON_ENV_PATH);
+		keysToRenmove.add(IDFEnvironmentVariables.IDF_PATH);
+		keysToRenmove.add(IDFEnvironmentVariables.ESP_IDF_VERSION);
+		keysToRenmove.add(IDFEnvironmentVariables.IDF_CCACHE_ENABLE);
+		keysToRenmove.add(IDFEnvironmentVariables.IDF_COMPONENT_MANAGER);
+		keysToRenmove.add(IDFEnvironmentVariables.IDF_MAINTAINER);
+		keysToRenmove.add(IDFEnvironmentVariables.OPENOCD_SCRIPTS);
+		
+		for (String key : keysToRenmove)
+		{
+			if (environment.containsKey(key))
+			{
+				environment.remove(key);
+			}
+		}
+	}
 }
```

Comment:
Instead of cleaning up system environment variables, why can't we rely only on the environment variables configured in the IDE and that includes PATH as well? 

IDFEnvironmentVariables().getEnvMap()

Otherwise, if we go with the cleanup approach, we'll have to maintain this code all the time whenever there are changes in the IDF environment variables.

Please check?